### DISCUSSION TITLE
Disable warnings

### DISCUSSION THREAD

**Discussion by Autraz** [nuxt/nuxt#26196] [Questions]
Hi is it possible to disable some or at least all warnings from the server console?

I tried to override a vue warnHandler with this plugin

```ts
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.config.warnHandler = () => null
})
```

unfortunatelly it's not working, any ideas?

**Comment by RohanReddy2Cool** [comment]
```
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.config.errorHandler = (err: unknown, instance: ComponentPublicInstance | null, info: string) => {
    console.warn("ERROR: ", err, info);
  }

  nuxtApp.vueApp.config.warnHandler = (msg: string, instance: ComponentPublicInstance | null, trace: string) => {
    if (msg.startsWith('Failed to resolve component: Mjx')) {
      // TODO: Ignore these for now
      return;
    }
    if (msg.includes('mjx-container')) {
      // TODO: Ignore these for now
      return;
    }
    console.warn("WARN: ", msg, trace);
  }
})
```
worked for me. Hope that helps.

**Reply by SparK-Cruz** [reply]
![image](https://github.com/user-attachments/assets/07a4f40b-1acc-4610-9638-0d8e2ca4cc2a)
For some reason when the code gets here the stack length is zero and instance is null. So it can't see our custom warnHandler and falls into the "else" logic which calls the default warning handler.

This is what the brave/chromium debugger sees.
